--- gem2rpm-0.8.1/lib/gem2rpm/distro.rb~	2013-04-24 10:31:39.000000000 +0300
+++ gem2rpm-0.8.1/lib/gem2rpm/distro.rb	2013-04-24 10:34:23.957321979 +0300
@@ -1,6 +1,7 @@
 module Gem2Rpm
   class Distro
     FEDORA = :fedora
+    PLD = :pld
     OPENSUSE = :opensuse
     DEFAULT = :default
 
@@ -22,6 +23,8 @@
         end
       elsif !release_files.grep(/SuSe/).empty?
         OPENSUSE
+      elsif !release_files.grep(/pld/).empty?
+        PLD
       else
         DEFAULT
       end
