--- gem2rpm-0.9.2/lib/gem2rpm/helpers.rb~	2013-04-29 23:19:15.000000000 +0300
+++ gem2rpm-0.9.2/lib/gem2rpm/helpers.rb	2013-04-29 23:24:07.509077652 +0300
@@ -24,7 +24,7 @@
     # '<' pair.
     def self.expand_pessimistic_requirement(requirement)
       next_version = Gem::Version.create(requirement.last).bump
-      return ['=>', requirement.last], ['<', next_version]
+      return ['>=', requirement.last], ['<', next_version]
     end
 
     # Expands the not equal version operator '!=' into equivalent '<' and
